lib/finch.ex

```diff
@@ -165,6 +165,11 @@ defmodule Finch do
 
   @doc """
   Builds an HTTP request to be sent with `request/3` or `stream/4`.
+
+  When making HTTP/1.x requests, it is possible to send request body in a streaming fashion.
+  In order to do so, the `body` parameter needs to take form of a tuple `{:stream, body_stream}`,
+  where `body_stream` is a [Stream](https://hexdocs.pm/elixir/Stream.html). This functionality
```

Comment:
I think there may be a simpler way to link to the Stream module without the need to explicitly link. I am afk right now, but I'll take a look when I am back. Otherwise I think these docs are suitable 👍

Reply:
ok